### DISCUSSION TITLE
Multi-Language Support: don't reply with all languages

### DISCUSSION THREAD

**Discussion by baboulebou** [openid/authzen#26] [General]
I don't think it's a good idea to reply with all language strings, this could be a huge response and it would probably slow-down the PDP too. Instead, have the client PEP request the language it wants its responses in, and just return that 1 reason string.
==> This requires adding a Language parameter to the request calls.

**Comment by ogazitt** [comment]
seems like the entire language mechanism ought to be optional... and if it's specified at all, it should rely on a more generic "options" mechanism which includes a bag of key-value pairs.  it would be a perfect example of how a PDP/PEP interaction can become richer by specifying a "language": "es" option, which would instruct the PDP to localize any human-readable reason strings to that locale.

**Comment by tulshi** [comment]
Converted to issue #32
